Troubleshooting assist with local URL (#35372)
home-assistant/home-assistant.github.io: source/voice_control/troubleshooting.markdown

     ![Create alias for entity name](/images/assist/assist_create_alias.png)
5. If you just ask "What is the weather" when you have forecast entities for multiple entities, Assist always returns the data for the place that was first added. Currently, there is no way to change that.

## My voice assistant understands me, processes the command, but I do not get back a voice response

> The voice response is generated in Home Assistant by one of our supported text-to-speech (or {% term TTS %}) engine.

Reviewer: ```suggestion
The voice response is generated in Home Assistant by one of our supported text-to-speech (or {% term TTS %}) engines.
```

The voice assistant device then fetches the audio file from Home Assistant and plays it back.

For this fetching process to work, Home Assistant must communicate its own URL to the device.
If you have a complex network setup, or if you changed this setting in the past, the URL communicated could be wrong.

> This is how you can fix it *(note: this process requires "Advanced mode" located in your user's setting)*

Reviewer: ```suggestion
To fix the URL, follow these steps:
```

> 1. Go to [Setting > System > Network](https://my.home-assistant.io/redirect/network/)

Reviewer: ```suggestion
1. In {% my profile title="your user profile" %}, enable **Advanced Mode**.
2. Go to {% my network title="**Settings** > **System** > **Network**" %}.
```

> 2. Change your Local Network Home Assistant URL to a URL that can be reached locally and that points to Home Assistant

Reviewer: ```suggestion
2. Change your Local Network Home Assistant URL to a URL that can be reached locally and that points to Home Assistant
  - For most users, the **Automatic** option works and is recommended.
```

   ![Create alias for entity name](/images/assist/local_url.png)

> Please note that, for most users, the automatic option works and is recommended.

Reviewer: ```suggestion
```

     ![Create alias for entity name](/images/assist/assist_create_alias.png)
5. If you just ask "What is the weather" when you have forecast entities for multiple entities, Assist always returns the data for the place that was first added. Currently, there is no way to change that.

> ## My voice assistant understands me, processes the command, but I do not get back a voice response

Reviewer: ```suggestion
## I don't get a voice response

My voice assistant understands me and processes the command, but I don't get a voice response.
```
Do you think we can shorten the title, and provide the details in the intro para? It makes the title easier to read in the side nav. (yes, I know. Some of the other titles are long, too :sweat_smile: )

     ![Create alias for entity name](/images/assist/assist_create_alias.png)
5. If you just ask "What is the weather" when you have forecast entities for multiple entities, Assist always returns the data for the place that was first added. Currently, there is no way to change that.

## My voice assistant understands me, processes the command, but I do not get back a voice response

The voice response is generated in Home Assistant by one of our supported text-to-speech (or {% term TTS %}) engines.
The voice assistant device then fetches the audio file from Home Assistant and plays it back.

For this fetching process to work, Home Assistant must communicate its own URL to the device.
If you have a complex network setup, or if you changed this setting in the past, the URL communicated could be wrong.

To fix the URL, follow these steps:

1. In {% my profile title="your user profile" %}, enable **Advanced Mode**.
2. Go to {% my network title="**Settings** > **System** > **Network**" %}.

> 2. Change your Local Network Home Assistant URL to a URL that can be reached locally and that points to Home Assistant

Reviewer: ```suggestion
3. Change your Local Network Home Assistant URL to a URL that can be reached locally and that points to Home Assistant
```
